player/src/App.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { Typography, Container, makeStyles, Paper, Grid, Button } from '@material-ui/core';
+import { Synth } from "tone";
+
+const useStyles = makeStyles((theme)=>({
+  root: {
+    width: '100%',
+    maxWidth: 500,
+    flexGrow: 1,
+  },
+  paper: {
+    padding: theme.spacing(2),
+    textAlign: 'center',
+    color: theme.palette.text.secondary,
+  },
+}));
+
+var synth = new Synth().toMaster();
+
+function handleClickQ(e) {
+  e.preventDefault();
+  console.log('The link was clicked.');
+  synth.triggerAttackRelease("C4", "8n");
+}
+function handleClickW(e) {
+  e.preventDefault();
+  console.log('The link was clicked.');
+  synth.triggerAttackRelease("D4", "8n");
+}
+function handleClickE(e) {
+  e.preventDefault();
+  console.log('The link was clicked.');
+  synth.triggerAttackRelease("E4", "8n");
+}
+function handleClickU(e) {
+  e.preventDefault();
+  console.log('The link was clicked.');
+  synth.triggerAttackRelease("F4", "8n");
+}
+function handleClickI(e) {
+  e.preventDefault();
+  console.log('The link was clicked.');
+  synth.triggerAttackRelease("G4", "8n");
+}
+function handleClickO(e) {
+  e.preventDefault();
+  console.log('The link was clicked.');
+  synth.triggerAttackRelease("A4", "8n");
+}
```

Comment:
There is probably an opportunity to condense your code and make it easier to update in the future. You could make a component called like MusicButton. Each MusicButton would have an attribute called note that would equate to the note you want to play. You could pass the attribute note down through props and have one event listener for the component.